kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -29,11 +29,16 @@
   import { now } from 'kolibri.utils.serverClock';
   import { ContentNodeResource, ContentNodeSearchResource } from 'kolibri.resources';
   import router from 'kolibri.coreVue.router';
+  import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
   import { events, MessageStatuses } from 'hashi/src/hashiBase';
   import { validateTheme } from '../../utils/themes';
   import { PageNames } from '../../constants';
   import ContentModal from './ContentModal';
 
+  const allButTopicTypes = Object.keys(ContentNodeKinds).filter(
+    k => k === ContentNodeKinds.TOPIC
+  );
```

Comment:
I think this won't work, the API expect to receive a [comma separated list](https://github.com/endlessm/kolibri/blob/release-v0.14.x/kolibri/core/content/api.py#L192-L201) of content kind.

And we should also use the `Object.values`, because the keys are in uppercase and didn't match the kind in all cases.

This should b e something like:

```
  const allButTopicTypes = Object.values(ContentNodeKinds).filter(k => k === 'topic').join(',');
```

Reply:
You are right about the values! And the filter should be for `!==` instead of `===`. 

There is no need for converting to a string of comma separated values, because the serialization takes care of that already.